src/site/content/_redirects.yaml

```diff
@@ -237,3 +237,9 @@ redirects:
 # Redirect for the renamed WebTransport API.
 - from: /quictransport
   to: /webtransport
+
+# Redirect for consolidated Lighthouse audit for a11y names.
+- from: /aria-toggle-field-name/
+  to: /aria-name/
+- from: /aria-input-field-name/
+  to: /aria-name/
```

Comment:
I can't remember if it affects things, but I noticed we always leave the trailing slash off of the redirects so maybe we should remove them from here as well.